src/main/groovy/life/qbic/variantstore/parser/EnsemblParser.groovy

```diff
@@ -58,21 +67,22 @@ class EnsemblParser {
         for (final Gff3Feature feature : reader.iterator()) {
             if(feature.type.contains("gene")) {
                 numberOfGenes++
-                def gene = new Gene()
-                gene.geneId = feature.ID.split(":")[-1]
-                gene.bioType = feature.getAttribute("biotype")
-                gene.chromosome = feature.contig
-                gene.geneStart = feature.start
-                gene.geneEnd = feature.end
-                gene.strand = feature.strand
-                gene.symbol = feature.name
-                gene.version = feature.getAttribute("version") != null ? feature.getAttribute("version").toInteger(): -1
+
+                String geneId = feature.ID.split(":")[-1]
+                def bioType = feature.getAttribute("biotype")
+                def chromosome = feature.contig
+                def geneStart = feature.start as BigInteger
+                def geneEnd = feature.end as BigInteger
+                def strand = feature.strand.toString()
+                def symbol = feature.name
+                def version = feature.getAttribute("version") != null ? feature.getAttribute("version").toInteger(): -1
```

Comment:
You could think about extracting this mapping into a seperate method so it's easier to adapt and track in the future

Reply:
Noted for the next refactoring. Thanks 👍